hazelcast/src/main/java/com/hazelcast/util/MD5Util.java

```diff
@@ -38,7 +38,7 @@ public static String toMD5String(String str) {
         try {
             MessageDigest md = MessageDigest.getInstance("MD5");
             if (md == null || str == null) {
-                return "NULL";
+                return null;
```

Comment:
That change can cause NPEs in code pieces where we use this util. Can we check that?